content/Guides/agent-data-handling.mdx

```diff
@@ -227,6 +227,73 @@ The `Attachment` object has the following properties:
 - `contentDisposition`: The content disposition of the attachment which is either `inline` or `attachment`. Defaults to `attachment`.
 - `data`: The `DataType` of the attachment.
 
+## Sending Email Replies
+
+Both SDKs support sending replies to incoming emails using the `sendReply` method. This requires the email-auth-token to be present in the request metadata.
+
+### JavaScript SDK
+
+<CodeExample js={`import type { AgentContext, AgentRequest, AgentResponse } from '@agentuity/sdk';
+
+export default async function Agent(
```

Comment:
when rendered it looks like maybe we're using tabs inside these instead of spaces for the example code. can you make sure we use spaces for tab stops and use like 3 spaces.